### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by domoritz**
- add missing filed to support dates properly
- remove dead code
- export schema.aggregate

Partially addresses https://github.com/vega/polestar/issues/348

### CODE REVIEW

File: src/schema/schema.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import {legend} from './fielddef.legend.schema';
 // TODO(#620) refer to vega schema
 // var vgStackSchema = require('vega/src/transforms/Stack').schema;
 
+export {aggregate} from './fielddef.schema';
```
Comment (kanitw) [inline src/schema/schema.ts]: better create `vl.aggregate` in `aggregate.ts` and
- move  `AGGREGATE_OPS`  to `vl.aggregate.OPS`
- move `supportedEnums` and `supportedTypes` to `vl.aggregate`

Comment (domoritz) [inline src/schema/schema.ts]: Will be done in separate pr.
